plugins/jobs/src/js/components/JobStopRunModal.js

```diff
@@ -0,0 +1,86 @@
+import { Confirm } from "reactjs-components";
+import PropTypes from "prop-types";
+import React from "react";
+import ModalHeading from "#SRC/js/components/modals/ModalHeading";
+
+class JobStopRunModal extends React.Component {
+  getContentHeader(selectedItems, selectedItemsLength) {
+    let headerContent = ` ${selectedItemsLength} Job Runs`;
+    if (selectedItemsLength === 1) {
+      headerContent = "this";
+    }
+
+    return (
+      <ModalHeading key="confirmHeader">
+        {`Are you sure you want to stop ${headerContent}?`}
+      </ModalHeading>
+    );
+  }
+
+  getConfirmTextBody(selectedItems, selectedItemsLength) {
+    let bodyText;
+
+    if (selectedItemsLength === 1) {
+      bodyText = `the job run with id ${selectedItems[0]}`;
+    } else {
+      bodyText = "the selected job runs";
+    }
+
+    return <span key="confirmText">You are about to stop {bodyText}.</span>;
+  }
+
+  getModalContents() {
+    const { selectedItems } = this.props;
+    const selectedItemsLength = selectedItems.length;
+
+    return (
+      <div className="text-align-center">
+        {this.getConfirmTextBody(selectedItems, selectedItemsLength)}
+      </div>
+    );
+  }
+
+  render() {
+    const { onClose, open, selectedItems } = this.props;
+    let rightButtonText = "Stop Job Run";
+    const selectedItemsLength = selectedItems.length;
+
+    if (selectedItems.length > 1) {
```

Comment:
I would actually go with ternary here. Ternary sends a clear message that the value could be either or. In this case default value and this logic are even broken in space

Reply:
Can look into that. I am trying to champion a non-multiline-ternary rule. So if I can express it here in a single line. Sure.

Reply:
Any specific reason for championing except of a taste?

Reply:
There is a larger discussion here: https://jira.mesosphere.com/browse/DCOS-38625.

I personally find readability to be the major issue as the markers for conditional are `?` and `:`, over multiline it gets hard to see there is a condition.